dcap/types/src/tcb.rs

```diff
@@ -71,13 +78,41 @@ type SgxExtensions = Vec<AttributeTypeAndValue>;
 /// The TCB info provided by the PCK(Provisioning Certification Key) leaf
 /// certificate
 #[derive(Debug, PartialEq)]
-pub struct PckTcbInfo {
-    svns: [u32; 16],
+pub struct TcbInfo {
```

Comment:
Renamed, the Pck prefix I had used when I was developing in the attestation repo, to disambiguate from the TCB data that comes from <https://api.trustedservices.intel.com/sgx/certification/v4/tcb?fmspc={}>
Up for suggestions on the "info" part, maybe it needs to go away.